docs/conformance.md

```diff
@@ -1,14 +1,44 @@
 # Conformance testing
 
+The Conformance tests are subset of our e2e test suite that defines set of features
+defining Kubevirt core functionality. The goal of the suite is to make sure that selected features work across different Vendors, CNIs, CSIs, CRIs implementations.
+
+## How are Conformance tests selected
+
+By default tests with a `[Conformance]` tag (and from release 0.59 also decorator) will be executed and tests with a
```

Comment:
How about adding the label decorator?

Reply:
What do you mean by adding?

Reply:
I mean, here we are only mentioning the `[Conformance]` tag.I think we want to switch using the Conformance label decorator.

Reply:
Right, as we don't have the label yet I would leave this out and fill in later.

Reply:
Part of another PR: https://github.com/kubevirt/kubevirt/pull/12682/files#diff-2be6ff8bee7a7b41764cd824e3e7afbc1ae3facd2893bdeb60f7ac978303a387R8